docs/developer/conventions/code/log.rst

```diff
@@ -156,4 +157,4 @@ Log4J offers similar functionalities with a wider range of handler implementatio
 
 We are not claiming that Java logging in superior to Log4J, neither we are forcing peoples to use Java logging. We push for usage of Java logging API, which may very well redirect to Log4J under the hood through java.util.logging.Log4JLoggerFactory implementations.
 
-Commons-logging is widely used in server containers, but other communities like scientists face a different picture. For example the NetCDF library developed by the University Corporation for Atmospheric Research (UCAR) uses SLF4J, yet other logging framework that aims to be a replacement for commons-logging.
\ No newline at end of file
+Commons-logging is widely used in server containers, but other communities like scientists face a different picture. For example the NetCDF library developed by the University Corporation for Atmospheric Research (UCAR) uses SLF4J, yet other logging framework that aims to be a replacement for commons-logging.
```

Comment:
Looks like you just did a newline at the end, although that should ready "yet another logging framework".

Reply:
Which it turns out I can fix.

Reply:
Not sure what happened there, maybe a newline correction by the online GH editor, my "fix" is the extra blank line 43, before the table starts

Reply:
so +1 for fixing another thing